source/admin_manual/obox/index.rst

```diff
@@ -203,13 +203,16 @@ If multiple backends do changes to the same mailbox at the same time, Dovecot wi
 
 .. code-block:: none
 
-   doveadm metacache pull -u user@domain --latest-only --clean 10.0.0.5
+   doveadm -o plugin/metacache_index_merging=none force-resync -u user@domain INBOX
+
 
-.. versionadded:: v2.4;v3.0 To allow easier migration of users and to support the new needs brought up with
-   the :ref:`dovecot_cluster_architecture` the ``doveadm metacache pull``
```

Comment:
What was wrong with this versionadded here? I don't like the standalone versionadded, since you never know what scope it means, and whether it means the text above or below. Could this be in its own section with a title? Then the versionadded could be below title and it would be clearer what its scope is.

Reply:
I just changed it because it is not rendered at all currently. 

I agree that it is clearer when it is prefixed with the versionadded part.  I changed the commit to be just about reorder the code-blocks and fix the formatting so it is properly rendered again.